pkg/taskutil/taskutil.go

```diff
@@ -96,11 +96,15 @@ func NewTask(ctx context.Context, client *containerd.Client, container container
 		if len(args) != 2 {
 			return nil, errors.New("parse logging path error")
 		}
-		ioCreator = cio.TerminalBinaryIO(u.Path, map[string]string{
+		parsedPath := u.Path
+		// For Windows, remove the leading slash
+		if (runtime.GOOS == "windows") && (strings.HasPrefix(parsedPath, "/")) {
```

Comment:
Is this the right place to remove the extra slash?

From the report `"binary:///C:/Program Files/nerdctl.exe?_NERDCTL_INTERNAL_LOGGING=C%3A%5CProgramData%5Cnerdctl%5C052055e3"` it seems it has 3 slashes in front of the file path: `binary:///`

Where is that generated and why does it have three slashes?

Reply:
The path is prefixed with a "/" in containerd/containerd [LogURIGenerator()](https://github.com/containerd/containerd/blob/a406da962818ba9ca8f38b4ac5215753ceb1846c/pkg/cio/io.go#L299C2-L308C3). See the comment in [LogURIGenerator()](https://github.com/containerd/containerd/blob/a406da962818ba9ca8f38b4ac5215753ceb1846c/pkg/cio/io.go#L299C2-L308C3).

Reply:
ah interesting, it is expected that url.parse returns `/c:/path` https://github.com/golang/go/commit/844b625ebcc7101e09fb87828a0e71db942a2416

This actually feels like a bug in `cio.TerminalBinaryIO` when it is checking abs path logic but I there might be some nuance that I am missing.